client/src/views/Search.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import type { Theme } from '@mui/material/styles';
+import FormControl from '@mui/material/FormControl';
+import { NavLink } from 'react-router-dom';
+import Select from '@mui/material/Select';
+import InputBase from '@mui/material/InputBase';
+import SearchIcon from '@mui/icons-material/Search';
+import Divider from '@mui/material/Divider';
+import MenuItem from '@mui/material/MenuItem';
+import { SelectChangeEvent } from '@mui/material';
+// import NativeSelect from '@mui/material/NativeSelect';
+import Avatar from '@mui/material/Avatar';
+const useStyles = makeStyles((theme: Theme) => ({
+  input: {
+    marginLeft: theme.spacing(3),
+    flex: 10,
+  },
+  iconButton: {
+    padding: 10,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    backgroundColor: 'black',
+    borderRadius: '25px',
+    '&:hover': {
+      backgroundColor: 'inheret',
+      borderRadius: '10px',
+    },
+  },
+  searchBar: {
+    fontFamily: 'DM Sans',
+    display: 'flex',
+    position: 'relative',
+    flexDirection: 'row',
+    background: 'white',
+    borderRadius: '40px',
+    width: '100%',
+  },
+  formControl: {
+    margin: theme.spacing(2),
+    minWidth: 90,
+    borderRadius: '40px',
+    borderColor: 'pink',
+    notched: 'false',
+  },
+}));
+function Search() {
+  const classes = useStyles();
+  const [selectedSearchCategory, setSelectedSearchCategory] = React.useState<string>('');
+  const [searchText, setSearchText] = React.useState<string>('');
+
+  const selectSearchCategory = (event: SelectChangeEvent<string>) => {
+    setSelectedSearchCategory(event.target.value as string);
+  };
+  return (
+    <div className={classes.searchBar}>
+      <FormControl variant="standard" sx={{ borderRadius: '20px' }}>
+        <Select
```

Comment:
i'm seeing a bit of weird behavior here, in that the arrow next to the select is not clickable. similar to the [mui docs](https://mui.com/material-ui/react-select/), the entire area should be part of the drop down